src/slic3r/GUI/Tab.cpp

```diff
@@ -6079,8 +6085,9 @@ void Page::update_visibility(ConfigOptionMode mode, bool update_contolls_visibil
 #ifdef __WXMSW__
     if (!m_show) return;
     // BBS: fix field control position
-    wxTheApp->CallAfter([this]() {
-        for (auto group : m_optgroups) {
+    auto groups = this->m_optgroups;
```

Comment:
@womendoushihaoyin 
Is this change intended? Any reason to change it?

Reply:
Yes, this change is necessary. If "this" is captured, it will cause unexplained occasional crashes.